From 71c35fd527021e8da2cc00c19830b3bef86bcd1a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20Hertzog?= <hertzog@debian.org>
Date: Sun, 11 Oct 2015 11:34:00 +1100
Subject: Disable creation of _sources directory by Sphinx

 We do this to save some space as the sources of the documentation
 are not really useful in a binary package.
 .
 This is a Debian specific patch.
Forwarded: not-needed
Origin: vendor

Patch-Name: 02_disable-sources-in-sphinxdoc.diff
---
 docs/conf.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 7707cf3..f179438 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -200,7 +200,10 @@ html_additional_pages = {}
 #html_split_index = False
 
 # If true, links to the reST sources are added to the pages.
-#html_show_sourcelink = True
+html_show_sourcelink = False
+
+# Do not ship a copy of the sources
+html_copy_source = False
 
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
 #html_show_sphinx = True
